_posts/en/newsletters/2019-10-30-newsletter.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Bitcoin Optech Newsletter #70'
+permalink: /en/newsletters/2019/10/30/
+name: 2019-10-30-newsletter
+slug: 2019-10-30-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter requests help testing release candidates,
+describes discussions about simplified LN commitments using CPFP
+carve-out, and summarizes several top-voted questions and answers from
+the Bitcoin StackExchange.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Help test release candidates:** experienced users are encouraged to
+  help test the latest release candidates for the upcoming versions of
+  [Bitcoin Core][Bitcoin Core 0.19.0] and [C-Lightning][c-lightning
+  0.7.3].  <!-- FIXME: verify latest state Tuesday night -->
+
+## News
+
+- **LN simplified commitments:** in two separate threads, developers
+  from LND discussed their work on implementing simplified commitments,
```

Comment:
'developers from Lightning Labs' or 'developers of LND' (Lightning Labs is the company, LND the product)

Reply:
That's an important distinction, thank you for mentioning it.  I'll edit.